draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -395,7 +395,13 @@ does not guarantee that the next request will be successful. Server metrics may
 conditions like the one shown in the example from {{service-limit}}.
 
 A server MAY return `RateLimit` response fields independently
-of the response status code.  This includes throttled responses.
+of the response status code.
+This includes throttled responses.
+
+Servers should be careful in returning `RateLimit` fields in
+redirection responses (eg. 3xx status codes) because
+a low `RateLimit-Remaining` value could refrain the client
+from issuing requests.
```

Comment:
```suggestion
from issuing requests.
For example, a client could not follow the redirection in the response below
for at least 10 seconds because  `RateLimit-Remaining` is 0.

~~~ http-message
HTTP/1.1 301 Moved Permanently
Location: /foo/123
RateLimit-Remaining: 0
RateLimit-Limit: 10
RateLimit-Reset: 10

~~~
```

Reply:
@darrelmiller @unleashed WDYT?

Reply:
This got me thinking that it does not make much sense for clients to consider rate limiting information to retry a redirect endpoint even if it was temporary - I can only think of questionable cases in which you could use the headers to stop redirection cycles or (ab)using the mechanism for having the same endpoint redirect to different locations each time someone performs a request.

From this text I'd only change the "For example, a client could not follow ..." with "For example, given the below rate limiting fields, a client would not immediately follow ..." to emphasize that it's not that it cannot follow the redirection right away, but that given the information it is expected (but not mandatory) that the client will wait until the time window is over.

Reply:
@unleashed PTAL